packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -104,9 +103,18 @@ export interface TableOptions {
   /**
    * Whether server-side encryption with an AWS managed customer master key is enabled.
    * @default - server-side encryption is enabled with an AWS owned customer master key
+   *
+   * @deprecated - This property is deprecated, use encryption instead
    */
   readonly serverSideEncryption?: boolean;
 
+  /**
+   * Whether server-side encryption with an AWS managed customer master key is enabled.
+   * @default - server-side encryption is enabled with an AWS owned customer master key
+   *
+   */
+  readonly encryption?: TableEncryption;
```

Comment:
```suggestion
  readonly encryption?: tableEncryption;
```

Reply:
This is coming from enum `TableEncryption`